cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,12 @@
-/// <reference types="cypress" />
-
 describe('Sign In page', () => {
+  const email = 'mlemeshev@i.ua';
+  const password = 'User1234';
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
+    cy.get('input[type="email"]').type(email);
+    cy.get('input[type="password"]').type(password);
+    cy.get('.btn').click();
+    cy.get('#__next > nav > div > ul > li:nth-child(4) > a')
+      .should('contain', 'mlemeshev');
```

Comment:
The selector '#__next > nav > div > ul > li:nth-child(4) > a' is quite brittle and can easily break with minor changes to the page structure. Consider using a more stable method to select elements, such as by adding data-test attributes to your elements.